spec/e2e/transaction_spec.rb

```diff
@@ -23,4 +23,37 @@
       expect(a.thing).to eq(b)
     end
   end
+
+  describe 'transaction behaviour when a validation fails' do
+    let(:clazz) do
+      UniqueClass.create do
+        include Neo4j::ActiveNode
+        property :name
+        validates :name, presence: true
+      end
```

Comment:
using `stub_active_node_class` is the new, better way, to do this:

```ruby
stub_active_node_class('Clazz') do
  property :name
  validates :name, presence: true
end
```

Then you would refer to `Clazz` instead of `clazz`